ftp/1.6/modules/ROOT/partials/common-file-lock.adoc

```diff
@@ -0,0 +1,10 @@
+== File Locks
+File system level locks are not possible in FTP or SFTP, so each operation uses a Mule lock that protects only the file from other flows in the same Mule application. These locks do not protect the file from external processes. However, if the Mule application is running on a cluster, the lock is distributed.
+
+The file lock is automatically released when any of the following occurs:
```

Comment:
Change this to:
"The file lock is automatically released when either of the following occur:"